.kitchen.dokken.yml

```diff
@@ -16,36 +16,32 @@ verifier:
 platforms:
 - name: debian-7
   driver:
-    image: debian:7
+    image: dokken/debian-7
     pid_one_command: /sbin/init
     intermediate_instructions:
       - RUN /usr/bin/apt-get update
-      - RUN /usr/bin/apt-get install apt-transport-https lsb-release procps net-tools -y
+      - RUN /usr/bin/apt-get install -y init-system-helpers
```

Comment:
Actually this looks like it's in the current image https://github.com/someara/dokken-images/blob/master/debian-7/Dockerfile#L7